100_Numpy_exercises_with_hints_with_solutions.md

```diff
@@ -355,8 +354,7 @@ print(np.trunc(Z))
 `hint: np.arange`
 
 ```python
-Z = np.zeros((5,5))
-Z += np.arange(5)
+Z = np.tile(np.arange(0, 5), (5,1))
```

Comment:
Can you also leave the previous solution such that people can see the correspondence between the two solutions ?